doc/src/package.rst

```diff
@@ -619,6 +598,18 @@ input.  Not all features of LAMMPS support OpenMP threading via the
 USER-OMP package and the parallel efficiency can be very different,
 too.
 
+.. note::
+
+   If you build LAMMPS with the GPU, USER-INTEL, and / or USER-OMP
```

Comment:
What about KOKKOS? with OpenMP?

Reply:
Can KOKKOS be used in combination with these packages?

Reply:
Theoretically it should be possible. Whether it is practical, I do not know since KOKKOS requires many kokkos-ized styles.

Reply:
@stanmoore1 Hi Stan, on the package doc page we have a warning about setting the number of OMP threads when using multiple packages. KOKKOS hasn't been included in this, but maybe should be. Can you comment on how this should be modified to include KOKKOS:

+   If you build LAMMPS with the GPU, USER-INTEL, and / or USER-OMP
+   packages, be aware these packages all allow setting of the *Nthreads*
+   value via their package commands, but there is only a single global
+   *Nthreads* value used by OpenMP.  Thus if multiple package commands are
+   invoked, you should insure the values are consistent.  If they are
+   not, the last one invoked will take precedence, for all packages.
+   Also note that if the :doc:`-sf hybrid intel omp command-line switch <Run_options>` is used, it invokes a "package intel" command, followed by a
+   "package omp" command, both with a setting of *Nthreads* = 0. Likewise
+   for a hybrid suffix for gpu and omp.

Reply:
@wmbrownIntel @akohlmey Sorry for the delay. I've never tried using KOKKOS with other packages, but the same should be true. It is also possible to invoke more than 1 package command for the same package, in which only the last setting will be used. We should also make sure that the user knows the package command will overwrite the `OMP_NUM_THREADS` env variable.